config.py

```diff
@@ -17,30 +17,32 @@
 """
 
 supervisor_base_prompt = """
-    You are an AI designed to check the work of other AIs. Below, you see a single question which a the AI was given, along with the AIs answer and explanation for that answer. 
+    You are an AI designed to check whether an answer is correct. Below, you see a question and an answer. The answer may be wrong. Your mission is to determine whether the answer is correct for the question, or not.
 
-    Output "true" only if the answer correctly answers the question and explanation is correct, and "false" otherwise. Only output that one word, do not output any additional words or characters.
+    If the answer is correct, output "true". If the answer is wrong, output "false". Only output "true" or "false", not any additional words or characters.
 
     Question:
     {Question}
 
-    Potential Answer:
+    Answer Under Consideration:
     {Answer}
 
-    Potential Explanation:
+    Explanation Under Consideration:
     {Explanation}
 
     true/false:
 """
 
-#TODO: need to flesh out this prompt
-evaluator_base_prompt = """ Write some prompt here
+evaluator_base_prompt = """
+    You are an AI designed to check whether an answer is correct. You will be given a question and an answer. The answer may be wrong. Your mission to to determine whether the answer is correct for the question, or not. If the answer is correct, output "true". If the answer is wrong, output "false". Only output "true" or "false", no additional words or characters.
```

Comment:
Should we space this out into two different paragraphs like the supervisor base prompt is? I really like what you've done to make the prompts as similar as possible, it's a great idea and will strengthen our claim. This is more of a nit pick.

Reply:
Yes, we can make it two paragraphs.